packages/esm-patient-list-app/src/patient-list-detail/patient-list-detail.component.tsx

```diff
@@ -38,19 +39,23 @@ interface PatientListDetailProps {
   patientListUuid: string;
 }
 
-const PatientListDetailComponent: React.FC<RouteComponentProps<PatientListDetailProps>> = ({ match }) => {
+const PatientListDetailComponent: React.FC<RouteComponentProps<PatientListDetailProps>> = () => {
   const patientListUuid = getPatientListUuidFromUrl();
   const { t } = useTranslation();
   const [currentPage, setPageCount] = useState(1);
   const [currentPageSize, setCurrentPageSize] = useState(10);
   const [searchString, setSearchString] = useState('');
-  const { data: patientListDetails } = usePatientListDetails(patientListUuid);
+  const { data: patientListDetails, mutate: patientListDetailsMutate } = usePatientListDetails(patientListUuid);
   const { data: patientListMembers } = usePatientListMembers(
     patientListUuid,
     searchString,
     (currentPage - 1) * currentPageSize,
     currentPageSize,
   );
+  const [editPatientListDetail, showOverlay] = useState(false);
```

Comment:
This naming of these two destructured variables is a bit weird (the convention is `const [foo, setFoo] = ...`). It would be great if you could rename them - that makes it much easier for others to understand that these are typical state variables.

Reply:
Sure! I should've kept it in mind.
Thanks for picking it up!